controller_manager/scripts/spawner.py

```diff
@@ -179,4 +180,6 @@ def main(args=None):
 
 
 if __name__ == '__main__':
-    sys.exit(main())
+    warnings.warn(
+        "'spawner.py' is deprecated, please use 'spawner' (without .py extension)", DeprecationWarning)
```

Comment:
```suggestion
        "'spawner.py' is deprecated, please use 'spawner' (without .py extension)",
        DeprecationWarning,
    )
```

Reply:
@Timple If you are sending a PR from an organization and not your personal account, please make sure to tick "allow edits from maintainers". It's enabled by default AFAIk but not from organization forks.

I did this change locally and validated with flake8, I hope I managed to copy the right hting over, this should fix the flake8 failure on the CI

Reply:
Unfortunately the tick-box is missing. Otherwise I would have enabled it. Not sure why it's not there.

Reply:
Ugh, it's apparently a pretty big issue: https://github.com/isaacs/github/issues/1681